message/router/middleware/instant_ack.go

```diff
@@ -2,6 +2,7 @@ package middleware
 
 import "github.com/ThreeDotsLabs/watermill/message"
 
+// InstantAck makes the handler instantly acknowledge the incoming message, regardless of any errors.
```

Comment:
some information that it can be useful for increasing throughput, when order and at least once delivery is ok for us

Reply:
4c18a1ce2169c218bc64caa54c3d6875f248758c